synapse/models/inet.py

```diff
@@ -48,17 +46,28 @@ def getDataModel():
             ('inet:mac',  {'subof':'str', 'regex':'^([0-9a-f]{2}[:]){5}([0-9a-f]{2})$', 'lower':1, 'nullval':'??',
                            'ex':'aa:bb:cc:dd:ee:ff','doc':'A 48 bit mac address'}),
 
-            ('inet:netuser',  {'subof':'sepr','sep':'/','fields':'site,inet:fqdn|user,inet:user',
+            ('inet:netuser',{'subof':'sepr','sep':'/','fields':'site,inet:fqdn|user,inet:user',
                                'doc':'A user account at a given web address','ex':'twitter.com/invisig0th'}),
 
-            ('inet:netmesg',  {'subof':'sepr','sep':'/','fields':'site,inet:fqdn|from,inet:user|to,inet:user|sent,time',
-                               'doc':'A message sent from one user to another within a web community',
-                                'ex':'twitter.com/invisig0th/gobbles/20041012130220'}),
+            ('inet:netpost',{'subof':'sepr', 'sep':'@', 'fields':'netuser,inet:netuser|time,time',
+                             'doc':'A post made by a netuser at a specific time','ex':'twitter.com/invisig0th/20150302225401'}),
+
+            ('inet:netgroup',   {'subof':'sepr','sep':'/','fields':'site,inet:fqdn|name,ou:name','doc':'A group within an online community'}),
+
+            ('inet:netmemb',    {'subof':'comp','fields':'user,inet:netuser|group,inet:netgroup'}),
+            ('inet:follows',  {'subof':'comp','fields':'src,inet:netuser|dst,inet:netuser'}),
```

Comment:
Need to sort out / select terms for 'src' vs 'dst' - still find this confusing and can see different analysts getting this backwards.